src/test/suite/tokenizer.test.ts

```diff
@@ -60,13 +67,53 @@ const tests: TestCase[] = [
   ["aåäöb", ["aåäöb"]],
 ];
 
+const cssDialectTokenizerTests: LanguageTokenizerTests[SupportedLanguageId] = {
+  tests: [
+    ["min-height", ["min-height"]],
+    ["-webkit-font-smoothing", ["-webkit-font-smoothing"]],
+    ["(min-width: 400px)", ["(", "min-width", ":", "400px", ")"]],
+    ["prefers-reduced-motion", ["prefers-reduced-motion"]],
+  ],
+  // Leave kebab and dashes to css language specific tests.
+  exclusionPredicate: (input: string) => !!input.match("-"),
+};
+
+const languageTokenizerTests: LanguageTokenizerTests = {
+  css: cssDialectTokenizerTests,
+  scss: cssDialectTokenizerTests,
+};
+
 suite("tokenizer", () => {
   tests.forEach(([input, expectedOutput]) => {
-    test(input, () => {
-      const output = tokenize(input, (match) => match[0]);
+    test(`tokenizer test, input: ${input}`, () => {
+      const output = tokenize(input, "anyLang", (match) => match[0]);
```

Comment:
Aha this is why you need the typecast in `tokenize`.  Hmm.  I don't think that our supported languages for parser and supported languages for tokenizer need to be the same, so I wonder if that's even the right type to use?  Not quite sure what the right answer is here 🤔

Maybe we should have a type `LanguageId` that is a superset of `SupportedLanguageId`?  And then we could include `plaintext`, and use that here?

cc/ @auscompgeek @AndreasArvidsson